lib/net/validations.rb

```diff
@@ -119,6 +119,12 @@ def self.normalize_ip6(ip)
       IPAddr.new(ip, Socket::AF_INET6).to_s rescue ip
     end
 
+    def self.normalize_network_address(ip, mask)
+      IPAddr.new(ip).mask(mask).to_s
+    rescue
```

Comment:
Should this log something? Probably at the info level.

Reply:
Done.